components/ContactUs/ContactUs.jsx

```diff
@@ -1,45 +1,44 @@
-import labels from '@/constants/labels.en'
 import styles from '@/styles/ContactUs.module.css'
 import { Icon } from '@/components'
 
 const ContactUs = (props) => {
-    const { id } = props
+    const { id,title,chat,descriptionChat,malito,email,malito1,email1,basedin,city,visit,instagramlink,linklinkedin,linkfacebook } = props
```

Comment:
Please an array of objects to pass the menu props (see the main menu as reference)

Reply:
Sorry, But I did not understand. The main menu you mean index?

Reply:
done